docs/howto/danceable-lyrics-input-tool/README.md

```diff
@@ -0,0 +1,485 @@
+# Danceable Lyrics
+In this guide, we use the [Alteryx Python SDK](https://pypi.org/project/ayx-python-sdk/) and [Alteryx Plugin CLI](https://pypi.org/project/ayx-plugin-cli/) to create tool that connects directly to a set of data files and determines the most "danceable" songs containing a set of lyrics. We will be utilizing [Polars](https://www.pola.rs/) for fast data processing.
+
+# Download Input Data
+Download and extract the required dataset files. You will reference them later on in this guide. We provide to sets of this data -- the full and a truncated, smaller version.
+
+* The full version expands to ~11 GiB of data and [can be downloaded here](https://drop.alteryx.com/public/file/SAlShV5VqkOI5_Jyx1me3w/Danceable_Lyrics_demo_data-Inspire-2023.7z).
+* A smaller truncated version that expands to ~5.7 GiB of data [can be downloaded here](https://drop.alteryx.com/public/file/yHv77vLzKkCXHj8-OWVg0Q/Danceable_Lyrics_demo_data-Inspire-(Truncated)-2023.7z).
+
+See the `README.md` within the respective archives for source information.
```

Comment:
TW review:

Go to the `README.md` files within the respective archives for source information.